### ISSUE TITLE
PDF Manual Sources

### ISSUE THREAD

**Issue by tresf** [LMMS/lmms.io#117]
Placeholder for finding a home for the PDF manual "source" (i.e. Word/Publisher/Etc).

@Lukas-W, @sti2nd, @Umcaruje, @stakeoutpunch do you have any opinions on this?  I think it would be beneficial to have a copy.

Here's some history of our email conversation.
<br><br>
<hr>

> **Narayan**,

> It is your right to keep any creative works you have created (and to provide the publication and sources under a license you deem appropriate), so I don't want to sound pushy, however, we do a great job of keeping all other creative sources which the project relies on (at least those resources we have sources for) backed up into an online repository which gives the project the security of knowing the source will outlive the creator.

> So it's less a matter of **who** will support the manual moving forward, but more of an **if** it is no longer maintained, and when it comes to a project the and magnitude of ours, that often means **when** it is no longer maintained.

> But there are some other benefits to this approach as well... As our wiki maintainers find issues with either the online or PDF version of our documentation, we can track it in our tracker (either open a ticket, or use the comments/highlight feature on the source document).  Furthermore, GitHub has version control -- albeit not tailored for publishing -- can revert to an earlier state of a document if needed.

> On a personal note -- as one of the technical reviewers of the Packt book and author of 500+ technical documents -- I can reassure you that very few people edit documents they haven't created themselves, but there are some benefits in committing the source of your work upstream, so I wanted to leave you with at least these thoughts (i.e. no pressure, just sharing some thoughts) -@tresf

<hr>

> **Tres,**
> Yes I do agree that keeping the sources in a repository has these advantages.  How do I place my source docs in repository? Could you please point to the process? -**Narayan**

**Comment by Umcaruje** [comment]
So what is happening with this manual initiative?

@raindropsfromsky was some progress made on updating the manual?

**Comment by raindropsfromsky** [comment]
HI

I will need one more week to make the latest version of LMMS manual.

After that, I can make the odt file available to anyone who wants to
translate.
(The last version was translated to German).

But I am confused as regards licensing options.
So I will need some time to decide.
Till then, the current arrangement should be OK.

Regards,
Narayan

**Comment by Umcaruje** [comment]
>I will need one more week to make the latest version of LMMS manual.

>After that, I can make the odt file available to anyone who wants to
translate.

:+1: awesome!

**Comment by tresf** [comment]
@raindropsfromsky please license the work as you see fit.  If the license allows us to alter or build on top of your work under certain guidelines, well that would be most beneficial to the project in the event you become unavailable.

**Comment by Umcaruje** [comment]
@raindropsfromsky any progress on this issue?

**Comment by raindropsfromsky** [comment]
Sorry about responding. Yes, it's nearly through. One more week...

**Comment by tresf** [comment]
@raindropsfromsky sribd has what appears to be an illegal copy of this up on their site.  Can you issue a takedown notice?  I'd do it, but you haven't transferred ownership yet.

http://www.scribd.com/doc/144754824/Lmms-Manual#scribd

To issue a takedown notice:

https://www.scribd.com/copyright/report-infringement

**Comment by raindropsfromsky** [comment]
Hi

First of all, the original manual is not a commercial venture, and does not
generate revenue for me or the LMMS project.

I am not familiar at all with Scribd, but it looks like it allows each
member to share non-CR material.
No one seems to be making any money.
So if the manual is shared there, it does not cause a loss to me or the
LMMS project.

So my first reaction is, "why not?"

My manuals are already posted publicly at the product's own website.
IINW this should prevent anyone else from selling it as his own work.

In fact, I felt that such indirect hosting should be encouraged, so that
more and more people get to know about LMMS.

The only problem is that if the manual is updated, such indirect hosts
would not update them!

Still if this is considered as an undesired activity, I can authorize any
entity to take action on my behalf.

******
There is yet another issue also: The take-down notice requires me to
declare that I am the exclusive owner of the work.

But I am not sure of my copyrights in the first place.

Is there a difference in how a manual, a novel and a patent are protected?

Specifically, does "prior work" affect the copyrights of a manual, as in
case of a patent?

   1. While creating the book, I go through the existing material, such as
   wiki. I also get my doubts clarified from users and coders. Do they have
   any claim on the book, too?

   2. While I do not copy from the existing wiki, my manual is bound to
   resemble the wiki partly, because both of them have similar screenshots and
   description about the controls.

   After all, how many different ways are there to describe a volume, pan
   or gain knob?

   But technically, can any wiki writer claim that the manual has
   reproduced some substance from the wiki?


Regards,
Narayan

**Comment by tresf** [comment]
> So my first reaction is, "why not?"

Because they charge money to download it when it should only be offered for free.  If you don't care, neither do I, TL;DR it's your work.

**Comment by raindropsfromsky** [comment]
Oh, do they? Then it is certainly objectionable!

So how do I authorize you? Or do we have an official representative at the
LMMS project for such purposes?

Regards
Narayan

**Comment by tresf** [comment]
Since this document is yours, I can't per:

From Scribd:

 * You are the creator of content posted to Scribd without your permission You are the duly authorized representative of the rights holder of content posted to Scribd without permission
 * You may not use this form to report infringements of works for which you do not hold a copyright. If you suspect that another person’s work has been posted without permission, please contact the rights holder or their publisher.

Last we discussed ownership, you were pretty clear.

> In other words,  I would be giving the source for safekeeping,  not giving it away.

So it's still yours.

**Comment by raindropsfromsky** [comment]
Done!

On Thu, Nov 19, 2015 at 12:43 PM, Tres Finocchiaro <notifications@github.com

**Comment by tresf** [comment]
> Done!

:+1:

**Comment by tresf** [comment]
Back on topic of the manual... are we going to commit the source documents to the project tree or close this out?  It's entirely your call @raindropsfromsky.

**Comment by raindropsfromsky** [comment]
The code repository approach won't work with my odt ->pdf conversation
method. Each version of odt is a separate entity, and we cannot apply the
version control below that level.

Therefore, I will keep posting the latest pdf, and send the complete odt
file to anyone who wants to translate.

Planning to finish the manual in the coming week. Will need a few
volunteers to proofread.

Regards
Narayan

**Comment by tresf** [comment]
> The code repository approach won't work with my odt ->pdf conversation
method. Each version of odt is a separate entity, and we cannot apply the
version control below that level.

I was more asking in regards to the sources of it.  Are you keeping them private or would you like them shared?  If you are keeping them private and keeping ownership, there's no need to leave this thread open.

In regards to placing the source code under version control, I'm not sure how Google Doc would handle the layout, but it would give an audit trail and version without installing 3rd party authoring software.  Just food for thought.

If we go this route, LMMS does maintain a Google Account which we could use for adding collaborators, but the decision of how to share this work is solely up to you.  If it is shared with us, then we can help with the technical portions.

**Comment by Wallacoloo** [comment]
**If** you want version control for the next major manual revision, you
could consider porting it to LaTeX, which is a text format that easily
compiles to pdf, dvi, etc.

**Comment by tresf** [comment]
> easily compiles to pdf, dvi, etc.

Right, but it can be a bear to author.  I think that was proposed once once before.

**Comment by raindropsfromsky** [comment]
yes, I tried two  popular LaTex editors but could not adjust to it at all.

I do not maintain sources separately. It is usually a odp where I assemble
the screnshots with annotations. In simpler cases I just take the
screenshots and paste it in the odt file directly.

So on the whole, let's go ahead with the arrangements I outlined in my last
mail.

BTW I got reply from SCRIBD. They have blocked the copy posted there...


Regards,
Narayan
